scraper/wgScraper.py

```diff
@@ -23,18 +25,27 @@
 db = client['plagiarismDB']
 windCol = db['windCollection']
 
+windCol.create_index("date", expireAfterSeconds=(2 * 86400))  # 2 days
+
 while True:
     browser.get(wgSite)
 
-    wait.until(ec.visibility_of_element_located((By.XPATH, "//span[@class='wgs_wind_avg_value']")))
+    wait.until(ec.visibility_of_element_located(
+        (By.XPATH, "//span[@class='wgs_wind_avg_value']")))
 
-    avg = float(browser.find_element_by_xpath("//span[@class='wgs_wind_avg_value']").text)
-    low = float(browser.find_element_by_xpath("//span[@class='wgs_wind_min_value']").text)
-    high = float(browser.find_element_by_xpath("//span[@class='wgs_wind_max_value']").text)
-    direction = browser.find_element_by_xpath("//span[@class='wgs_wind_dir_value']").text
+    avg = float(browser.find_element_by_xpath(
+        "//span[@class='wgs_wind_avg_value']").text)
+    low = float(browser.find_element_by_xpath(
+        "//span[@class='wgs_wind_min_value']").text)
+    high = float(browser.find_element_by_xpath(
+        "//span[@class='wgs_wind_max_value']").text)
+    direction = browser.find_element_by_xpath(
+        "//span[@class='wgs_wind_dir_value']").text
 
-    print(f"{{ avg: {avg}, low : {low}, high : {high}, direction : {direction} }}")
+    timestamp = datetime.datetime.utcnow()
 
-    windCol.insert_one({ "avg": avg, "low" : low, "high" : high, "direction" : direction})
+    windCol.insert_one(
+        {"date": timestamp, "avg": avg, "low": low, "high": high, "direction": direction}
+    )
 
-    time.sleep(1)
\ No newline at end of file
+    time.sleep(1)
```

Comment:
Maybe change to 900 seconds (15 minutes). Do you think so is a good period to update the data? Or do we want to live data always? That would be cool, but maybe quite expensive.

Reply:
I am busy writing the logic that will schedule the scraper to run once every 15 mins. Just wanted to write it in a new branch since this one(in this PR) was created with the intent to add db cleaning. 

My new branch is for configuring the scheduled times so the change you are talking about will be in my next PR.

Reply:
![bellissimo](https://user-images.githubusercontent.com/32794346/80369203-2bcfe580-888e-11ea-928d-a00a38eb0f85.jpg)